src/connection/xmpp/handlers/presence.ts

```diff
@@ -98,7 +101,7 @@ export default class extends AbstractHandler {
          description: 'from ' + jid.bare,
          type: NOTICETYPE.contact,
          fnName: NOTICEFUNCTION.contactRequest,
-         fnParams: [jid.bare]
+         fnParams: [jid.bare, nickname]
```

Comment:
Is this used somewhere?

Reply:
It is used to keep the nickname we received from the presence request in the cache,  we later use it in the dialogs/contact.ts file.